otaserver/coap.py

```diff
@@ -6,12 +6,18 @@
 import aiocoap.resource as resource
 
 from aiocoap import Context, Message, CONTENT
+from functools import reduce
 
 logger = logging.getLogger("otaserver")
+COAP_PORT = 5678
```

Comment:
I prefer keeping the default CoAP port and use the application command line option